src/create_disk.sh

```diff
@@ -90,7 +93,7 @@ case "$arch" in
         -n ${BOOTPN}:0:+384M -c ${BOOTPN}:boot \
         -n 2:0:+127M -c 2:EFI-SYSTEM -t 2:C12A7328-F81F-11D2-BA4B-00A0C93EC93B \
         -n 3:0:+1M   -c 3:BIOS-BOOT  -t 3:21686148-6449-6E6F-744E-656564454649 \
-        -n ${ROOTPN}:0:0     -c ${ROOTPN}:root       -t ${ROOTPN}:0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        -n ${ROOTPN}:0:${rootfs_size}     -c ${ROOTPN}:root       -t ${ROOTPN}:0FC63DAF-8483-4772-8E79-3D69D8477DE4
```

Comment:
Hmm, should these be prepended by a `+`? Otherwise from a scan of `sgdisk(8)`, it seems like it interprets it as an offset from the start of the disk (so the partition size would actually be `rootfs_size - (384 + 127 + 1)`).

Reply:
Yes, very nice catch!  I had that in the other PR but lost it when reworking it here.
Fixed :arrow_down: